src/pages/CreatePersona/InitialStep/CreatePersonaCard.tsx

```diff
@@ -1,8 +1,8 @@
 import React, { FC } from 'react';
+import { EntityCard } from 'components/CreateSpotAndPersona/CreateCard/EntityCard';
 import { useMutation, useQuery } from '@apollo/react-hooks';
-import { GET_CARD, GetCardType, UPDATE_CARD } from 'global/graphqls/SpotAndPersona';
-import { CardType } from 'global/graphqls/schema';
-import { CreateEntityCard } from 'components/CreateSpotAndPersona/CreateCard/CreateEntityCard';
+import { GET_CARD, GetCardType, UPDATE_CARD } from '../../../global/graphqls/SpotAndPersona';
```

Comment:
"global/graphqls/SpotAndPersona" is enough

Reply:
changed